pkg/controller/migcluster/migcluster_controller.go

```diff
@@ -123,7 +120,7 @@ func (r *ReconcileMigCluster) Reconcile(request reconcile.Request) (reconcile.Re
 		if errors.IsNotFound(err) {
 			return reconcile.Result{}, nil
 		}
-		log.Trace(err)
+		log.Trace(err) // TODO - handle with liberr
```

Comment:
@jortel what is the appropriate action to take in places like this where the err isn't immediately returned? Just drop the `log.Trace()`?

Reply:
In the main reconcile loop, I think we should leave this as is.  Same for anywhere else in the code that is Log.Trace(err) but not returning the error.